homeassistant/components/light/lutron_caseta.py

```diff
@@ -0,0 +1,81 @@
+"""Support for Lutron Caseta lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components.lutron_caseta import (
+    LUTRON_CASETA_DEVICES, LUTRON_CASETA_SMARTBRIDGE)
+from homeassistant.components.light.lutron import to_hass_level
+
+DEPENDENCIES = ['lutron_caseta']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup Lutron lights."""
+    devs = []
+    for _caseta_light in hass.data[LUTRON_CASETA_DEVICES]['light']:
+        dev = LutronCasetaLight(hass, _caseta_light,
+                                hass.data[LUTRON_CASETA_SMARTBRIDGE])
+        devs.append(dev)
+    add_devices(devs, True)
+
+    return True
+
+
+class LutronCasetaLight(Light):
+    """Representation of a Lutron Light, including dimmable."""
+
+    def __init__(self, hass, device_info, smartbridge):
+        """Initialize the light."""
+        self._prev_brightness = None
+        self._device_id = device_info["device_id"]
+        self._device_type = device_info["type"]
+        self._device_name = device_info["name"]
+        self._state = None
+        self._smartbridge = smartbridge
+        self.update()
```

Comment:
Hello thank for looking at the PR.  Caseta does support additional component like 'covers' and 'Thermostats'  I was hoping that I (or someone else) would add these additional component over time.  Would you still prefer that I still remove/merge the light component for now?